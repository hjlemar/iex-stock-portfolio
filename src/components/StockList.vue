<template>
        <v-list dense>
        <app-stock-list-item v-for='(stock,i) in stocks'
          :stock="stock"
          :key='i'></app-stock-list-item>
      </v-list>
</template>

<script>
import StockListItem from '@/components/stock/StockListItem';

export default {
  components: {
    appStockListItem: StockListItem,
  },
  props: ['stocks'],
};
</script>

