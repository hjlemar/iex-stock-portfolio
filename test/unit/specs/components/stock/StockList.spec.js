import { shallow, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import StockList from '@/components/stock/StockList';
import Vuetify from 'vuetify';

const localVue = createLocalVue();
localVue.use(Vuex);
localVue.use(Vuetify);

describe('StockList', () => {
  let store;

  beforeEach(() => {
    store = new Vuex.Store({ });
  });

  it('should render component and match snapshot2', () => {
    const wrapper = shallow(StockList, { store, localVue });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
