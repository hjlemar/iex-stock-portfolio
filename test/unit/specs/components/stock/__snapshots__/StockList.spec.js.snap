// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StockList should render component and match snapshot2 1`] = `undefined`;
